c2rust-analyze/src/rewrite/expr/mod.rs

```diff
@@ -14,6 +16,8 @@ pub fn gen_expr_rewrites<'tcx>(
     hir_body_id: BodyId,
 ) -> Vec<(Span, Rewrite)> {
     let mir_rewrites = mir_op::gen_mir_rewrites(acx, asn, mir);
-    let hir_rewrites = hir_op::gen_hir_rewrites(acx.tcx(), mir, hir_body_id, &mir_rewrites);
+    let unlower_map = unlower::unlower(acx.tcx(), mir, hir_body_id);
+    let rewrites_by_expr = distribute::distribute(acx.tcx(), unlower_map, mir_rewrites);
+    let hir_rewrites = convert::convert_rewrites(acx.tcx(), hir_body_id, rewrites_by_expr);
```

Comment:
nit: Can the functions just be imported and called directly?

Reply:
I generally prefer to import the module and call `module::function()` rather than import the function directly.  This makes it clear when the callee is in a different module.